e2e/pages/portal/review-and-submit-page/review-and-submit-page.ts

```diff
@@ -0,0 +1,64 @@
+import { type Locator, type Page } from '@playwright/test';
+import { ParcelsSection } from './parcels-section';
+import { PrimaryContactSection } from './primary-contact-section';
+import { OtherOwnedParcelsSection } from './other-owned-parcels-section';
+import { GovernmentSection } from './government-section';
+import { LandUseSection } from './land-use-section';
+import { TURProposalSection } from './tur-proposal-section';
+import { OptionalDocumentsSection } from './optional-documents-section';
+
+export class ReviewAndSubmitPage {
+  readonly page: Page;
+
+  // Locators
+  readonly submitButton: Locator;
+  readonly informationUseCheckbox: Locator;
+  readonly informationCorrectnessCheckbox: Locator;
+  readonly acceptVerificationCheckbox: Locator;
+  readonly submitDialog: Locator;
+  readonly finalSubmitButton: Locator;
+
+  // Sections
+  readonly parcelsSection: ParcelsSection;
+  readonly otherOwnedParcelsSection: OtherOwnedParcelsSection;
+  readonly primaryContactSection: PrimaryContactSection;
+  readonly governmentSection: GovernmentSection;
+  readonly landUseSection: LandUseSection;
+  readonly turProposalSection: TURProposalSection;
+  readonly optionalDocumentsSection: OptionalDocumentsSection;
+
+  constructor(page: Page) {
+    this.page = page;
+
+    // Locators
+    this.submitButton = page.getByRole('button', { name: 'Submit' });
+    this.informationUseCheckbox = page.getByRole('checkbox', {
```

Comment:
Definitely something that could be cleaned up in the future using test-ids or just by selecting all the checkboxes in the dialog.

Reply:
In general, this is something that is tricky to balance. Ideally, locators should use user-facing features, since this is supposed to mimic user interaction. However, sometimes it is cumbersome or next to impossible to do so, so test ID's can be a good option.